owncloudApp/src/main/res/values-ar/strings.xml

```diff
@@ -339,8 +339,7 @@
   <string name="placeholder_media_time">12:23:45</string>
   <string name="camera_picture_upload_on_wifi">تحميل الصور عبر الواي فاي فقط</string>
   <string name="camera_video_upload_on_wifi">تحميل الفيديوهات عبر الواي فاي فقط</string>
-  <string name="camera_upload_path">/تحميل الكاميرا</string>
-  <string name="confirmation_disable_camera_uploads_title">تأكيد</string>
+    <string name="confirmation_disable_camera_uploads_title">تأكيد</string>
```

Comment:
Indentation problem when deletion of above string in each strings file

Reply:
I removed it using the string editor, so I would say it's been properly indented (4 spaces) and the indentation problem (2 spaces) was already present in all the translations (maybe because of transifex?). We could indent all the strings.xml files but it's not this PR scope

Reply:
I meant to keep same format in every string as it was before.